reddit_liveupdate/public/static/css/liveupdate.less

```diff
@@ -1,3 +1,8 @@
+// This file contains all css that will be necessary to run /live.
+// liveupdate-public contains CSS that must be available cross site (for callouts/banners/etc)
+
+@import "liveupdate-public";
```

Comment:
if `liveupdate-public` is imported into the base CSS, is it necessary to re-import it here?